src/java.net.http/share/classes/jdk/internal/net/http/HttpClientImpl.java

```diff
@@ -1686,6 +1690,14 @@ BufferSupplier getSSLBufferSupplier() {
         return sslBufferSupplier;
     }
 
+    private Duration getIdleConnectionTimeoutProp() {
+        // Http 2 in prop name somewhere
+        String s = Utils.getNetProperty("jdk.httpclient.idleConnectionTimeout");
+        if (s != null)
+            return Duration.ofMillis(Long.parseLong(s));
```

Comment:
Should we catch any `NumberFormatException` here and just default to `null` duration (i.e. no idle timeout)?